config/config.go

```diff
@@ -0,0 +1,39 @@
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/go-yaml/yaml"
+)
+
+// Launcher is launcher entity struct
+type Launcher struct {
+	Version string `yaml:"version"`
+	Image   string `yaml:"image"`
+}
+
+// Config is successConfig entity struct
+type Config struct {
+	APIURL   string   `yaml:"api-url"`
+	StoreURL string   `yaml:"store-url"`
+	Token    string   `yaml:"token"`
+	Launcher Launcher `yaml:"launcher"`
+}
+
+// ReadConfig returns parsed successConfig file
+func ReadConfig(configPath string) (Config, error) {
+	buf, err := ioutil.ReadFile(configPath)
+	if err != nil {
+		return Config{}, fmt.Errorf("failed to read successConfig file: %v", err)
```

Comment:
What is `successConfig`?

Reply:
You are right. The message is strange. I will fix.